bimmer_connected/api/utils.py

```diff
@@ -107,7 +107,7 @@ def anonymize_response(response: httpx.Response) -> AnonymizedResponse:
     brand = response.request.headers.get("x-user-agent", ";").split(";")[1]
     brand = f"{brand}-" if brand else ""
 
```

Comment:
Add this function to check for a valid json string:

`def is_json(jsonStr):
    try:
        json.loads(jsonStr)
    except ValueError as e:
        return False
    return True`